### ISSUE TITLE
Transform All Images Format to webp

### ISSUE THREAD

**Issue by benhaynes** [directus/api#396]
#### :rotating_light: MIGRATED FROM REQUESTS.GETDIRECTUS.COM :rotating_light: 

# Transform Image Format on Upload
### :+1: = 1
*Created 7 months ago by @levimichael*
 

-----
### @levimichael – :+1: (7 months ago)
Regardless of what format the user uploads, have directus convert to .webp

This format offers great file size reduction.

**Comment by rijkvanzanten** [comment]
This particular use case could be added as a hook too

**Comment by WellingGuzman** [comment]
Exactly, I never though of it to be attached to any browser, but an user preference. (like Rijk says, it could be done using hook)
